.github/workflows/pr.yml

```diff
@@ -0,0 +1,47 @@
+name: sync zoekt
+on:
+  push:
```

Comment:
Can we add a manual dispatch and create a test PR from this? https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
would the intention be to then close the test PR as part of the same work flow?

Reply:
No need - you can just create it, and if nobody reviews it, it should just stay open right? That would be an effective test run perhaps